Source/SwiftLintFramework/Rules/Style/SwitchCaseSortRuleExamples.swift

```diff
@@ -0,0 +1,340 @@
+enum SwitchCaseSortRuleExamples {
+    // to see the triggering and correction subsequently
+    static let examples: (triggering: [Example], nonTriggering: [Example], corrections: [Example: Example]) = {
```

Comment:
I'd like to see examples where the `switch` statement is contained (indented) in a function. Also, some comments and whitespace should be used in cases to proof that the rewriter doesn't break or remove them.